app/scripts/services/routes.js

```diff
@@ -102,9 +102,17 @@ angular.module("openshiftConsole")
       return annotation(route, "openshift.io/host.generated") !== "true";
     };
 
+    var isOverviewAppRouteToDisplay = function(route){
```

Comment:
@rhamilto I realize you may have nothing to do with this, but gonna ask anyway :)

Any idea where this annotation name came from?  As far as I have understood from conversations, `annotations` shouldn't have the word `alpha` in them because they really can't change anyway.  Apparently this is a bad practice (meaning if we can discourage it before it gets out there, we probably should).

I'm not sure I'm a fan of the word `overview` in the name either.  While today we are using it to display extra routes on the overview, is it the best name long term?  Would we ever want to use this elsewhere?  Seems unnecessarily restrictive.